tests/test_utils.py

```diff
@@ -349,3 +349,14 @@ def test_table_packing_pack(
 
     for actual, expected in zip(unpacked, table_data_1):
         assert actual == expected
+
+
+def test_table_packing_give_default_values(
+    table_1_np_arrays: Dict[str, UnpackedArray],
+    table_field_info: TableFieldInfo,
+):
+    # We should have a complete table at the point of unpacking
+    table_1_np_arrays["TRIGGER"] = np.array([])
+
+    with pytest.raises(AssertionError):
+        table_to_words(table_1_np_arrays, table_field_info)
```

Comment:
Would be nice to do a simple check for some part of the expected message in the returned exception, e.g. look for the TRIGGER column name in the returned message. While there's currently only two possible places this exception could be raised, it's good future-proofing to look for as exact a failure as possible.

Reply:
Good point, that's done. I'll merge.